surefire-report-parser/src/main/java/org/apache/maven/plugins/surefire/report/TestSuiteXmlParser.java

```diff
@@ -177,9 +183,27 @@ public void startElement(String uri, String localName, String qName, Attributes
                         currentSuite.incrementNumberOfSkipped();
                         break;
                     case "flakyFailure":
+                        testCaseFlakyFailure = new ReportTestCase.FlakyFailure(
+                                attributes.getValue("message"), attributes.getValue("type"));
+                        currentSuite.incrementNumberOfFlakes();
+                        break;
                     case "flakyError":
+                        testCaseFlakyError = new ReportTestCase.FlakyError(
+                                attributes.getValue("message"), attributes.getValue("type"));
                         currentSuite.incrementNumberOfFlakes();
                         break;
+                    case "stackTrace":
+                        currentElement = new StringBuilder();
+                        parseContent = true;
+                        break;
+                    case "system-out":
+                        currentElement = new StringBuilder();
+                        parseContent = true;
+                        break;
+                    case "system-err":
+                        currentElement = new StringBuilder();
+                        parseContent = true;
+                        break;
```

Comment:
Both are still there?

Reply:
Ah removed the other part but forgot about these ones. Fixed.